public/html/settings.html

```diff
@@ -69,6 +78,38 @@ <h2>Trigger Initial Setup</h2>
           </p>
           <a href="https://mint.intuit.com/settings.event?filter=property&setupRobinhood=true" target="_blank" class="button">Trigger Initial Setup</a>
         </section>
+
+        <!-- Multiple Accounts -->
+        <section id="multipleAccounts" class="main">
+          <header class="major">
+            <h2>Multiple Accounts</h2>
+          </header>
+          <p>
+            If you would like to enable syncing for multiple accounts, you can enable it here. <br />
+            Note that this will require you to delete any propeties created before turning on this feature. <br />
+            Similarly, turning off this feature will require you to delete any properties created while it was on. <br />
+            Each row should be the name EXACTLY as it appears on top of the <a href="https://robinhood.com/account">Robinhood Accounts</a> page. <br />
+            Whichever account is logged in at the time of syncing will be updated if found. Only one account can update at a time.
+          </p>
+          <input type="checkbox" id="setting-multipleAccountsEnabled" name="setting-multipleAccountsEnabled" />
+          <label for="setting-multipleAccountsEnabled">Allow Multiple Accounts</label>
+
+          <div class="mutiple-accounts-wrapper">
+            <div id="mutiple-account-rows"></div>
+            <template id="mutiple-accounts-input-wrapper-template">
+              <div class="mutiple-accounts-input-wrapper">
+                <label>
+                  RobinHood Account Name
+                  <input required type="text" class="setting-multipleAccounts-robinHoodInput" />
```

Comment:
Rather than having the user manually enter their account names, we can just scrape the account name every time we sync Robinhood. If the multiple account setting is not enabled, the value is ignored.

If it is enabled, we do a setup check (similar to the initial setup), where we check if this device has synced with the Robinhood account logged in. (These could be stored in an array in chrome.storage.sync). If the current device has not set up that account before, we can then trigger the property setup/check.

This would give users a more seamless experience, and reduce logic on the settings page since it would just be a checkbox.

Reply:
Alright, latest push should accomplish this... 
Need to probably test some of the flows and make sure the notifications seem correct in each one.

... Probably also need to add a delete script  under the debug at some point, more just for convienence, since adding and deleting all these accounts is crazy tedious.

Changing the `returnValue.accountName = accountNameElement.innerText;` to a static value works well enough for testing to add new accounts.